studybuddies/src/app/services/base-urlinterceptor.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { BaseURLInterceptorService } from './base-urlinterceptor.service';
+
+describe('BaseURLInterceptorService', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7202842)
```suggestion
describe("BaseURLInterceptorService", () => {
```